src/Slim/Middleware/Session.php

```diff
@@ -43,6 +43,7 @@ public function __construct($settings = [])
             'httponly'    => false,
             'name'        => 'slim_session',
             'autorefresh' => false,
+            'csrf'        => false
```

Comment:
Docs on  `slim/csrf` are [very clear](https://github.com/slimphp/Slim-Csrf#register-for-all-routes).
I also consider playing nice with that library would be awesome, but as I said it needs to rely on `class_exists()` rather than a preference.
Do you mind implementing this? 😄